data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1235 @@
+			`	You sit down, along with two other Quarg, and the rest of them leave the room with the door sliding shut behind them. Adrauni stares blankly at you for a while, then speaks. "I presume you understand what result came from my negotiations with the tedious Gegno."`
+			choice
+				`	"That they are upset with me?"`
+					goto "upset"
+				`	"That they will tolerate me?"`
+				`	"No, I do not."`
+					goto "no idea"
+
+			`	Adrauni continues to stare at you, then slowly blinks. "I see. What is your strange understanding of the term, tolerance, in this capacity?"`
+			choice
+				`	"Permission to land on their worlds."`
+				`	"Allow me access to their technology."`
+					goto "tech"
+				`	"I don't know."`
+					goto "dont know"
+
+			`	Adrauni ponders for a few moments. "That is a rather human response. Although we Quarg are, by your definition, minimally tolerative of your species, access to only some of our wondrous worlds we give you. However, what of humans to others? Should the Gegno, or species of other eccentricities, be given leisure to your planet Earth? May it be the case with you, though with anxious humanity, it certainly would not be." One of the other Quarg in the room says something to Adrauni, to which it begins a conversation in its own language.`
+			choice
+				`	"Is there no way I can work with the Gegno?"`
+					goto "work"
+				`	"I guess I should leave them alone."`
+					goto "alone"
+
+			label "tech"
+			`	Adrauni clicks its teeth together in what appears to be a laughing manner, a reaction you have yet to see from it. "And we many Quarg, of those who live among ambitious humans, allowance of our tech have we given? One is not owed rights to another's under pretensive tolerance." The other Quarg in the room look at Adrauni strangely, to which it begins a conversation with them in their own language. After a few moments, they too stare at you with strange looks, followed by an awkward silence, which is broken by Adrauni speaking to you once more. "With us as well, hardly do the Gegno share much intrigue at all."`
+			choice
+				`	"Then is there no way I can work with the Gegno?"`
+					goto "work"
+				`	"I guess I should leave them alone."`
+					goto "alone"
+
+			label "dont know"
+			`	"Indeed, you do not. To tolerate is to authorize unrestricted occurrence despite undesired preface. In this lies not the Gegno's interest, as once I have said before." It looks up in thought for several moments. "Perhaps in your naive judgment I am mistaken. Humans tolerate their own undesirables, those who have come to be known as corsairs. Portions of your species permit their tiresome actions, while others do not. Be that as it may, suppose a species other than humanity begins similar agitation. I would presume the case to be different." Adrauni then begins a conversation with the other Quarg in the room.`
```

Comment:
This would be under the knowledge that First Last *knows* about the Korath raids, which is not always the case, and I wouldn't give rebuttal or clarification of that here. If anything, perhaps the Quarg is intentionally not speaking of them, or is, but in the usual "Quarg pretending not to know" way.